DEVELOPER.md

```diff
@@ -15,7 +15,7 @@ build system. Our CI is based on [Github Actions](https://docs.github.com/en/act
 
 ### docker
 * Optional when you want to build a Docker image or run make inside Docker.
-* Recommened Version: 20.10.16
+* Recommended Version: 20.10.16
 * Installation Guide: https://docs.docker.com/engine/install/
 
 ### linters
```

Comment:
please also fill out the linters section with the exact versions and installation guides, thanks

Reply:
can you follow the format used for `make`, `docker` `go` etc, by deleting the `linters` header and adding a separate one
for `golangci-lint` , `yamllint` and `codespell` so each tools has an installation guide and version bullet point